support/notulen-queries.js

```diff
@@ -24,13 +24,13 @@ async function getZittingForNotulen(uuid) {
         besluit:behandelt ?agendapunten;
         besluit:isGehoudenDoor ?bestuursorgaanUri;
         besluit:geplandeStart ?geplandeStart;
-        prov:startedAtTime ?startedAt;
-        prov:endedAtTime ?endedAt;
         mu:uuid ${sparqlEscapeString(uuid)}.
       ?bestuursorgaanUri mandaat:isTijdspecialisatieVan ?mainBestuursorgaanUri.
       ?mainBestuursorgaanUri skos:prefLabel ?bestuursorgaanName.
       OPTIONAL {
-        ?uri prov:atLocation ?location.
+        ?uri prov:atLocation ?location;
+          prov:startedAtTime ?startedAt;
```

Comment:
probably best to wrap each of these in an optional statement as each of them can be missing